.vuepress/styles/index.scss

```
@@ -1,3 +1,7 @@
 .theme-container.custom-page-class .language-example {
   border: 4px solid #3eaf7c;
+}
+
+.home .hero img {
+  object-fit: scale-down;
 }
```

Could the icon in the Join Server button be changed to white/black instead?
```suggestion
}

.home .hero .external-link-icon {
  --external-link-icon-color: Canvas;
}
```

---

It'd be good to change the accent colour to something that fits Microsoft better — it was there in the old site, but was kinda broken and showed up only in light mode

```css
:root {
  --c-brand: rgb(0 94 183);
  --c-brand-light: rgb(0 94 183 / 90%);
}

html.dark {
  --c-brand: rgb(96 204 254);
  --c-brand-light: rgb(96 204 254 / 90%);
}
```

These are taken from the WinUI Figma file, check it out